### PR TITLE
Add Twisted to gallery

### PR DESCRIPTION
**Pull request by ofek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ You’ve come to the right place!
 - [*Gradio*](https://pypi.org/project/gradio/) ([`pyproject.toml`](https://github.com/gradio-app/gradio/blob/main/pyproject.toml))
 - [*pytermgui*](https://pypi.org/project/pytermgui/) ([`pyproject.toml`](https://github.com/bczsalba/pytermgui/blob/master/pyproject.toml))
 - [*stamina*](https://pypi.org/project/stamina/) ([`pyproject.toml`](https://github.com/hynek/stamina/blob/main/pyproject.toml))
+- [*Twisted*](https://pypi.org/project/twisted/) ([`pyproject.toml`](https://github.com/twisted/twisted))
```
Comment (ofek) [inline README.md:33]:
```suggestion
- [*Twisted*](https://pypi.org/project/twisted/) ([`pyproject.toml`](https://github.com/twisted/twisted/blob/trunk/pyproject.toml))
```

**Comment by hynek** [comment]
Thanks but I think I'll wait for a release, before linking to the PyPI page. 🤔

**Comment by ofek** [comment]
A release happened one week after I opened this, sorry I forgot to follow-up

**Comment by hynek** [comment]
yeah but they reverted it beforehand (I think because of -/_?). If you download the sdist, you'll see it's still setuptools. :|

**Comment by ofek** [comment]
Okay so waiting for the next release then I guess since they reverted that temporary revert https://github.com/twisted/twisted/blob/trunk/pyproject.toml

**Comment by ofek** [comment]
Soon... https://pypi.org/project/Twisted/23.8.0rc1/

**Comment by hynek** [comment]
I have rebased for a reason ;)

**Comment by hynek** [comment]
🎉

**Pull request merged**